recipes/cuda-nvcc/activate.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export NVCC_PREPEND_FLAGS_BACKUP="${NVCC_PREPEND_FLAGS}"
+export NVCC_PREPEND_FLAGS="${NVCC_PREPEND_FLAGS} -ccbin ${CXX}"
```

Comment:
Testing the package locally ( `conda install -c ..../staged-recipes/build_artifacts/ cuda-nvcc cuda-cudart` ) in a clean env.

This creates a broken compiler, due to `${CXX}` having no value. We need to either require a CXX compiler or only optionally add this env var when `CXX` exists.

```
(ctk) rmaynard@rmaynard-dt:~/Work/cmake/conda_test_build$ conda list
# packages in environment at /home/rmaynard/miniconda3/envs/ctk:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
cuda-cccl_linux-64        12.0.90              ha770c72_1    conda-forge
cuda-cudart               12.0.107             hcb278e6_0    file:///home/rmaynard/Work/conda-forge/staged-recipes/build_artifacts
cuda-cudart-dev           12.0.107             hcb278e6_0    file:///home/rmaynard/Work/conda-forge/staged-recipes/build_artifacts
cuda-cudart-dev_linux-64  12.0.107             hcb278e6_0    file:///home/rmaynard/Work/conda-forge/staged-recipes/build_artifacts
cuda-cudart-static        12.0.107             hcb278e6_0    file:///home/rmaynard/Work/conda-forge/staged-recipes/build_artifacts
cuda-cudart-static_linux-64 12.0.107             hcb278e6_0    file:///home/rmaynard/Work/conda-forge/staged-recipes/build_artifacts
cuda-nvcc                 12.0.76              h59595ed_0    file:///home/rmaynard/Work/conda-forge/staged-recipes/build_artifacts
cuda-version              12.0                 hffde075_0    conda-forge
libgcc-ng                 12.2.0              h65d4601_19    conda-forge
libgomp                   12.2.0              h65d4601_19    conda-forge
libstdcxx-ng              12.2.0              h46fd767_19    conda-forge
````